service/lib/agama/dbus/y2dir/manager/modules/Package.rb

```diff
@@ -39,6 +39,13 @@ def Available(_package_name)
       true
     end
 
+    # Determines whether the package is available
+    #
+    # @todo Perform a real D-Bus call.
+    def AvailableAll(_package_names)
+      true
```

Comment:
Can we always assume true?

Reply:
No. We can't. It's the same TODO we have for `Available` in the line above.

Reply:
But not that you mention it, I will search for the corresponding Trello card/Github issue/Whatever and will reference it here.

Reply:
Everything originated here (it says "_We need to add support for Package#Available and Package#Installed to D-Bus_):

Reply:
Done. Now we have https://trello.com/c/oRcZ5P5k/251-implement-missing-software-methods